src/runtime_src/core/tools/common/XBMain.cpp

```diff
@@ -66,6 +69,7 @@ void  main_(int argc, char** argv,
   // Hidden Options
   po::options_description hiddenOptions("Hidden Options");
   hiddenOptions.add_options()
+    ("device,d",    boost::program_options::value<decltype(sDevice)>(&sDevice)->default_value("")->implicit_value("default"), "If specified with no BDF value and there is only 1 device, that device will be automatically selected.\n")

```

Comment:
This addresses
`xbutil examine -d`
use case?

Reply:
Please see existing PR comments.

Reply:
Please see existing submission description.  First paragraph.